packages/@aws-cdk/aws-lambda/test/code-signing-config.test.ts

```diff
@@ -0,0 +1,107 @@
+      },
+      CodeSigningPolicies: {
+        UntrustedArtifactOnDeployment: lambda.UntrustedArtifactOnDeployment.WARN,
+      },
+    });
+  });
+
+  test('with multiple signing profiles', () => {
+    const signingProfile1 = new signer.SigningProfile(stack, 'SigningProfile1', { platform: signer.Platform.AWS_LAMBDA_SHA384_ECDSA });
+    const signingProfile2 = new signer.SigningProfile(stack, 'SigningProfile2', { platform: signer.Platform.AMAZON_FREE_RTOS_DEFAULT });
+    const signingProfile3 = new signer.SigningProfile(stack, 'SigningProfile3', { platform: signer.Platform.AWS_IOT_DEVICE_MANAGEMENT_SHA256_ECDSA });
+    new lambda.CodeSigningConfig(stack, 'CodeSigningConfig', {
+      signingProfiles: [signingProfile1, signingProfile2, signingProfile3],
+    });
+
+    expect(stack).toHaveResource('AWS::Lambda::CodeSigningConfig', {
+      AllowedPublishers: {
+        SigningProfileVersionArns: [
+          {
+            'Fn::GetAtt': [
+              'SigningProfile1D4191686',
+              'ProfileVersionArn',
+            ],
+          },
+          {
+            'Fn::GetAtt': [
+              'SigningProfile2E013C934',
+              'ProfileVersionArn',
+            ],
+          },
+          {
+            'Fn::GetAtt': [
+              'SigningProfile3A38DE231',
+              'ProfileVersionArn',
+            ],
+          },
+        ],
+      },
+      CodeSigningPolicies: {
+        UntrustedArtifactOnDeployment: lambda.UntrustedArtifactOnDeployment.WARN,
+      },
+    });
+  });
+
+  test('with description and with untrustedArtifactOnDeployment of "ENFORCE"', () => {
+    const platform = signer.Platform.AWS_LAMBDA_SHA384_ECDSA;
+    const signingProfile = new signer.SigningProfile(stack, 'SigningProfile', { platform });
+    new lambda.CodeSigningConfig(stack, 'CodeSigningConfig', {
+      signingProfiles: [signingProfile],
+      untrustedArtifactOnDeployment: lambda.UntrustedArtifactOnDeployment.ENFORCE,
+      description: 'test description',
+    });
+
+    expect(stack).toHaveResource('AWS::Lambda::CodeSigningConfig', {
+      AllowedPublishers: {
+        SigningProfileVersionArns: [{
+          'Fn::GetAtt': [
+            'SigningProfile2139A0F9',
+            'ProfileVersionArn',
+          ],
+        }],
+      },
+      CodeSigningPolicies: {
+        UntrustedArtifactOnDeployment: lambda.UntrustedArtifactOnDeployment.ENFORCE,
+      },
+      Description: 'test description',
+    });
+  });
+
+  test('import does not create any resources', () => {
+    const codeSigningConfigId = 'aaa-xxxxxxxxxx';
+    const codeSigningConfigArn = `arn:aws:lambda:::code-signing-config:${codeSigningConfigId}`;
+    const codeSigningConfig = lambda.CodeSigningConfig.fromCodeSigningConfigArn(stack, 'Imported', codeSigningConfigArn );
+
+    expect(codeSigningConfig.codeSigningConfigArn).toBe(codeSigningConfigArn);
+    expect(codeSigningConfig.codeSigningConfigId).toBe(codeSigningConfigId);
+    expect(stack).toMatchTemplate({});
```

Comment:
How about something like this instead?

```suggestion
    expect(stack).toCountResources('AWS::Lambda::CodeSigningConfig', 0);
```